/**:
  ros__parameters:
    intensity_threshold: 1.0
    existance_probability_threshold: 0.2
    filter_target_label:
      UNKNOWN : true
      CAR : false
      TRUCK : false
      BUS : false
      TRAILER : false
      MOTORCYCLE : false
      BICYCLE : false
      PEDESTRIAN : false
